python/hsml/engine/model_engine.py

```diff
@@ -352,18 +352,18 @@ def read_json(self, model_instance, artifact):
     def delete(self, model_instance):
         self._engine.delete(model_instance)
 
-    def add_tag(self, model, name, value):
+    def add_tag(self, model_instance, name, value):
```

Comment:
Just out of curiosity, why do you think `model_instance` is better?

Reply:
To make it consistent with the current naming when referring to a model in model_engine.py, for me it doesn't really matter if we call it model or model_instance, they are equal to me.

Reply:
Ok! To me `model_instance `is the same as `model`, but with an extra word. But it's fine for now